src/EnergyPlus/MixedAir.cc

```diff
@@ -4328,6 +4347,16 @@ namespace MixedAir {
 
 		}
 
+		if ( OAController( OAControllerNum ).CoolCoilFreezeCheck ) {
+			MaxOAFracBySetPoint = min( max( MaxOAFracBySetPoint, 0.0 ), 1.0 );
+			OAController( OAControllerNum ).MaxOAFracBySetPoint = MaxOAFracBySetPoint;
```

Comment:
@lgu1234 When I run the new CoolingCoilFreezingPrevention tets file, the output always shows zero for "Air System Outdoor Air Maximum Flow Fraction".  
Also, as we move towards OO structs it's bad to have a local variable with the same name as a struct variable.  Suggest changing the name slightly or just delete the local variable and use OAController( OAControllerNum ).MaxOAFracBySetPoint instead.  @Myoldmopar Is that the proper guidance?